api-catalog-services/src/main/resources/application.yml

```diff
@@ -246,6 +246,7 @@ eureka:
         metadata-map:
             apiml:
                 corsEnabled: true
+                corsAllowedOrigins: https://${apiml.service.hostname}:${apiml.service.port},${apiml.service.externalUrl}
```

Comment:
Actually, in theory the APIML could run in http

Reply:
This config applies to attls only, so I guess it's always https in this scenario